plugins/w3c_logs.yaml

```
@@ -0,0 +1,109 @@
+version: 0.0.1
+title: W3C
+description: File Input W3C Parser
```

I think we've been following the format of `Log parser for <technology>`
```suggestion
description: Log Parser for W3C
```